rasa/server.py

```diff
@@ -930,7 +930,8 @@ def _get_output_channel(
     ):
         requested_output_channel = tracker.get_latest_input_channel()
 
-    registered_input_channels = request.app.input_channels or []
+    # Interactive training does not set `input_channels`, hence we have to be cautious
+    registered_input_channels = getattr(request.app, "input_channels", None) or []
```

Comment:
```suggestion
    registered_input_channels = getattr(request.app, "input_channels", [])
```
Does this also work?

Reply:
it would, but I think `None` is more lightweight then `[]` . Also `[] or []` would evaluate to the last `[]`

Reply:
ah now i get it

Reply:
input channels itself could be `None` and I would like to avoid that

Reply:
ah good point, so if you use getattr and the attribute is assigned to none, it will return none? I think that's different from the functionality you'd get with .get() right?

Reply:
> I think that's different from the functionality you'd get with .get() right?

Nope thats the same

```
{"x": None}.get("x", 5) # returns None
```

Reply:
wild. TIL 😄